microsite/package.json

```diff
@@ -20,15 +20,16 @@
   },
   "prettier": "@backstage/cli/config/prettier",
   "resolutions": {
+    "@rspack/core": "^1.3.0",
     "node-polyfill-webpack-plugin": "^3.0.0"
   },
   "dependencies": {
-    "@docusaurus/core": "^3.1.1",
-    "@docusaurus/plugin-client-redirects": "^3.1.1",
-    "@docusaurus/preset-classic": "^3.1.1",
-    "@docusaurus/types": "^3.1.1",
+    "@docusaurus/core": "^3.7.0",
+    "@docusaurus/faster": "^3.7.0",
+    "@docusaurus/plugin-client-redirects": "^3.7.0",
+    "@docusaurus/preset-classic": "^3.7.0",
+    "@docusaurus/types": "^3.7.0",
```

Comment:
it would be safer to first upgrade to Docusaurus v3.7, and only then turn Docusaurus faster on in another PR

Doing many things in the same PR makes things harder to troubleshoot